Library/SRRecorderControl.m

```diff
@@ -82,6 +82,8 @@ typedef NS_ENUM(NSUInteger, _SRRecorderControlButtonTag)
     _SRRecorderControlMainButtonTag = 2
 };
 
+@interface SRRecorderControl () <NSAccessibility, NSViewToolTipOwner>
```

Comment:
It's unusual to add `NSAccessibility` conformance like this. What are you indicating?

Reply:
You might want `NSAccessibilityButton` which will populate the role for you